projects/client/RabbitMQ.Client/src/client/impl/SocketFrameHandler.cs

```diff
@@ -240,21 +240,21 @@ public void WriteFrameSet(IList<OutboundFrame> frames)
             var nbw = new NetworkBinaryWriter(ms);
```

Comment:
Could you tell why do we not write frames to `m_writer` directly instead of buffering them first? @michaelklishin @kjnilsson